etc/kytos/kytos.conf.template

```diff
@@ -32,6 +32,9 @@ listen = 0.0.0.0
 # openflow packets using TCP protocol. Default is 6633.
 port = 6633
 
+# The southbound protocol that will be used by the TCP server
+protocol_name = openflow
```

Comment:
@viniciusarcanjo we are trying to remove openflow 'dependency' from our core (we know that there is still some things to work on that sense). Because of that, I would rather prefer to leave the `'protocol_name'` default as an empty string. If someone is using the default openflow ports (6633 and 6653) this won't make any difference (considering this PR that improves our `known_ports` dict).

Reply:
Got it. I'll do.